docs/sql-ref-syntax-dml-insert-into.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: global
+title: INSERT INTO
+displayTitle: INSERT INTO
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+The `INSERT INTO` statement inserts new rows into a table. The inserted rows can be specified by value expressions, or resulted from a query.
+
+### Syntax
+{% highlight sql %}
+INSERT INTO [TABLE] [db_name.]table_name [partition_spec] value_clause | query
+
+partition_spec:
+    : (part_col_name1[=val1] [, part_col_name2[=val2], ...])
+
+value_clause:
+    : VALUES values_row [, values_row ...]
+
+    values_row:
+        : (val1 [, val2, ...])
+
+{% endhighlight %}
+
+### Examples
+{% highlight sql %}
+ CREATE TABLE employees (name VARCHAR(64), age INT, salary DECIMAL(9,2))
+   USING PARQUET PARTITIONED BY (age)
+
+ INSERT INTO employees
+   VALUES ('Amy Smith', 35, 200000.00)
+
+ INSERT INTO employees PARTITION (age = 35)
+   SELECT * FROM candidates WHERE name = "Bob Doe"
+{% endhighlight %}
```

Comment:
@huaxingao If we determine that we allow more syntax flavors .. could we please add one test for each ?